libraries/AP_HAL_ChibiOS/CanIface.cpp

```diff
@@ -582,32 +566,18 @@ void CANIface::pollErrorFlags()
 bool CANIface::canAcceptNewTxFrame(const AP_HAL::CANFrame& frame) const
 {
```

Comment:
we don't need the frame argument any more?

Reply:
Good point, removed. I chose to keep the argument to `select` as that's a much more invasive change. There are still some strange semantics surrounding it though especially on other HALs, but that's for a different day...